src/hott/06-contractible.rzk.md

```diff
@@ -331,8 +332,8 @@ The center of contraction in the based path space is `#!rzk (a , refl)`.
   ( is-contr-B : is-contr B)
   : is-contr (product A B)
   :=
-    ( (first is-contr-A , first is-contr-B) ,
-      \ p → path-product A B
+    ( ( first is-contr-A , first is-contr-B)
+    , \ p → path-product A B
               ( first is-contr-A) (first p)
               ( first is-contr-B) (second p)
               ( second is-contr-A (first p))
```

Comment:
this is also wrong and should be formatted similarly to my previous comment.

Reply:
Does that mean that there should **always** be a new line after the lambda arrow?

Reply:
No, only if the full lambda expression does not fit within a single line.

Reply:
E.g. `\ x → x` is totally fine, but `\ x → impossibly-long-expression-that-surpasses-80-character-limit` is not.